util/cron/nightlysubs.pm

```diff
@@ -117,7 +117,11 @@ sub endMailHeader {
 }
 
 sub endMailChplenv {
-    my $chplenv = `$chplhomedir/util/printchplenv --debug`;
+    my $ch = $chplhomedir;
```

Comment:
Consistency will lead to less confusion in the long run.  What are the advantages of using the relative path instead of the defined CHPL_HOME?  I'm fine with this patch going in as is, but the divergence worries me a little.